data-weave-lang-server/src/main/scala/org/mule/weave/lsp/WeaveLanguageServer.scala

```diff
@@ -235,7 +235,10 @@ class WeaveLanguageServer extends LanguageServer {
 
   @JsonNotification("weave/project/create")
   def createProject(): Unit = {
-    val projectProvider = new ProjectProvider(client, workspaceUri)
-    projectProvider.newProject()
+    CompletableFuture.supplyAsync(() => {
```

Comment:
Aren't we going to change this for a command?

Reply:
Yes, I'll change this next